pkg/kubectl/sorting_printer.go

```diff
@@ -60,8 +61,35 @@ func (s *SortingPrinter) sortObj(obj runtime.Object) error {
 	if len(objs) == 0 {
 		return nil
 	}
+
+	sorter, err := SortObjects(objs, s.SortField)
+	if err != nil {
+		return err
+	}
+
+	switch list := obj.(type) {
+	case *v1.List:
```

Comment:
Hrm, when are you in a v1 list?  Is this due to some other change that stopped decoding objects passed into here?